internal/provider/resources/flow.go

```diff
@@ -129,9 +128,6 @@ func (r *FlowResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 				Optional:    true,
 				Computed:    true,
 				Default:     listdefault.StaticValue(defaultEmptyTagList),
-				PlanModifiers: []planmodifier.List{
-					listplanmodifier.RequiresReplace(),
-				},
```

Comment:
This ensures changes to the resource no longer require replacement now that we implement Update.